src/ui/public/stringify/types/String.js

```diff
@@ -36,12 +36,14 @@ export default function _StringProvider(Private) {
     { id: false, name: '- none -' },
     { id: 'lower', name: 'Lower Case' },
     { id: 'upper', name: 'Upper Case' },
+    { id: 'title', name: 'Title Case' },
```

Comment:
This is not a comment for the pull author, but rather for us. We need to make this list pluggable